### PR TITLE
Parametrize DOCKERHUB_USER environment variable

### PR DESCRIPTION
**Pull request by konturn**
Meant to disambiguate staging and production docker-daily builds.

### CODE REVIEW

File: jenkins/JenkinsFile-rebuild-docker-images [hunk 1]
```diff
@@ -125,7 +125,7 @@ pipeline {
               ).trim()
 
               currentBuild.displayName = "${TVM_CURRENT_SHORT_REF}"
-              TVM_DOCKER_TAG = "${TIMESTAMP_TAG}-${TVM_CURRENT_SHORT_REF}"
+              TVM_DOCKER_TAG = "${TIMESTAMP_TAG}-${TVM_CURRENT_SHORT_REF}${params.tvm_docker_tag_suffix}"
```
Comment (areusch) [inline jenkins/JenkinsFile-rebuild-docker-images:128]: can you add a default to the param so the file runs if the param is not given?

Comment (konturn) [reply jenkins/JenkinsFile-rebuild-docker-images:131]: So I defined the parameter within the Jenkinsfile and added an empty default. Will the job fail to build if the parameter is undefined in the job configuration, or will the default be used?

Comment (areusch) [reply jenkins/JenkinsFile-rebuild-docker-images:131]: i am pretty sure the default will get used, let's try.

Comment (areusch) [reply jenkins/JenkinsFile-rebuild-docker-images:131]: can we actually make this a prefix? then we won't mix in staging with production

Comment (konturn) [reply jenkins/JenkinsFile-rebuild-docker-images]: yup--fixed

**Review by areusch — APPROVED** [review]

File: jenkins/JenkinsFile-rebuild-docker-images [hunk 2]
```diff
@@ -88,15 +88,16 @@ pipeline {
...
       DOCKERHUB_KEY = credentials('dockerhub-tlcpackstaging-key')
       DISCORD_WEBHOOK = credentials('discord-webhook-url')
   }
 
   parameters {
     string(name:"TVM_GIT_REV",
            defaultValue: "main",
-           description: 'Git revision to checkout.')
+           description: 'Git revision to checkout.'),
+    string(name:"DOCKERHB_USER", defaultValue: "tlcpackstaging",
```
Comment (areusch) [inline jenkins/JenkinsFile-rebuild-docker-images:99]: nit: DOCKERHUB

**Review by areusch — CHANGES_REQUESTED** [review]

**Review by areusch — APPROVED** [review]

**Review by leandron — APPROVED** [review]
Thanks! LGTM

**Pull request merged**